docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -1546,20 +1546,29 @@ Apple devices such as iPhones, iPads, iPods and Mac OS X (10.7+) support wireles
 
 Configure the feature
 ^^^^^^^^^^^^^^^^^^^^^
+NOTE: If EAP-TLS provisioning is desired, you have to configure a PKI before going any further. Two guides exists to assist you: 'PacketFence_PKI_Quick_Install_Guide', which covers PacketFence's implementation, or 'PacketFence_MSPKI_Quick_Install_Guide' which covers Microsoft's.
```

Comment:
Don't use the underscores; simply put the names of the guides.

`'PacketFence PKI Quick Install Guide`' ..